crates/sui-framework/packages/dwallet-system/sources/ethereum_state.move

```diff
@@ -126,10 +140,23 @@ module dwallet_system::ethereum_state {
         state.state_root
     }
 
-    public(friend) fun get_ethereum_state_network(
+    public(friend) fun set_latest_ethereum_state_id(
```

Comment:
why does this has to be a function? it should be only set once during the initialization of the state object

Reply:
delete unused function